ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -68,17 +79,41 @@ internal fun TipsTutorialLayout() = GodToolsTheme() {
         )
 
         Text(
-            text = stringResource(R.string.tutorial_tips_learn_headline),
+            text = stringResource(title),
             style = MaterialTheme.typography.titleLarge,
             textAlign = TextAlign.Center,
 
         )
 
         Text(
-            stringResource(R.string.tutorial_tips_learn_text),
-            style = MaterialTheme.typography.bodyLarge,
+            stringResource(body),
+            style = MaterialTheme.typography.bodyMedium,
             textAlign = TextAlign.Center,
             modifier = Modifier.padding(top = 16.dp)
         )
+
+        if (body2 != null) {
+
+            Text(
+                // Spacer(modifier = Modifier.minLinesHeight(minLines = 1, textStyle = MaterialTheme.typography.bodyMedium)),
+                stringResource(body2),
+                style = MaterialTheme.typography.bodyMedium,
+                textAlign = TextAlign.Center
+            )
+        }
+        if (anim == R.raw.anim_tutorial_tips_light) {
+            Button(
```

Comment:
the reason these are not currently pinned to the bottom is that they are in a `Column` that has `verticalArrangement = Arrangement.Center`. This is causing the Buttons to be vertically centered along with the rest of the content in this `Column`